integration_tests/test_catkin_shared_ros.py

```diff
@@ -20,9 +20,12 @@
 
 import integration_tests
 
+from snapcraft.tests import skip
```

Comment:
this is the wrong import!

Reply:
ah, pyramid is in the `integrationtests` `contrel` file. All good. Still faster to use the leaner version :wink:

Reply:
Ugh, you scared me. But yeah, as long as the tests pass, we'll clean that up soon.